ipysheet/easy.py

```diff
@@ -208,6 +208,15 @@ def cell_range(value, row_start=0, column_start=0, row_end=None, column_end=None
         A CellRange widget.
     """
     global _cells
+
+    try:
+        import numpy as np
```

Comment:
I think the `try except` should go away. It's not a big deal if ipysheet has NumPy for dependency, right? We can add it